src/vivarium_census_prl_synth_pop/constants/metadata.py

```diff
@@ -231,6 +231,9 @@
     "relation_to_household_head",
     "sex",
     "race_ethnicity",
+    "guardian_1",
+    "guardian_2",
+    "housing_type",
```

Comment:
Address and zipcode again here but I think merging into main would fix this anyways.

Reply:
Thanks for flagging, I see that you updated some lines near here in `main`, and I'll make a corresponding update in `observer.py` before merging.